app/models/sock_size.rb

```diff
@@ -1,4 +1,7 @@
 class SockSize < ApplicationRecord
   belongs_to :size
   belongs_to :sock
+
+  validates_presence_of :size, :sock, :quantity
```

Comment:
I don't think you need to validate for :size and :sock since the model already belongs to those.  It won't allow itself to be created without the association